AppKit/CPWindow/CPWindow.j

```diff
@@ -1958,6 +1967,80 @@ CPTexturedBackgroundWindowMask
 
         case CPLeftMouseDragged:
         case CPRightMouseDragged:
+            // First, we search for any tracking area requesting CPTrackingEnabledDuringMouseDrag.
+            // At the same time, we update the entered stack.
+            var mouseEnteredStack = [];
+
+            for (var i = 0; i < _trackingAreaViews.length; i++)
+            {
+                var aView          = [_trackingAreaViews objectAtIndex:i],
+                    trackingAreas  = [aView trackingAreas],
+                    convertedPoint = [aView convertPoint:point fromView:_windowView];
+                
+                if (![aView isHidden])
+                    
+                    for (var j = 0; j < trackingAreas.length; j++)
+                    {
+                        var aTrackingArea   = [trackingAreas objectAtIndex:j],
+                            trackingOptions = [aTrackingArea options];
+                        
+                        if (trackingOptions & CPTrackingEnabledDuringMouseDrag)
+                            
+                            if ((trackingOptions & CPTrackingActiveAlways) ||
+                                (trackingOptions & CPTrackingActiveInActiveApp) ||
+                                ((trackingOptions & CPTrackingActiveInKeyWindow) && ([self isKeyWindow])) ||
+                                ((trackingOptions & CPTrackingActiveWhenFirstResponder) && ([self isKeyWindow]) && (_firstResponder === aView)))
+                                
+                                if (CGRectContainsPoint(((trackingOptions & CPTrackingInVisibleRect) ? [aView visibleRect] : [aTrackingArea rect]), convertedPoint))
```

Comment:
Please wrap this if block in {}, it makes it much clearer that this is a dependent statement of the if above.